pkg/fanal/analyzer/language/elixir/mixlock/mixlock.go

```diff
@@ -0,0 +1,49 @@
+package mixlock
+
+import (
+	"context"
+	"os"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/hex/mix"
+
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&mixLockAnalyzer{})
+}
+
+const (
+	version = 1
+)
+
+// mixLockAnalyzer analyzes 'mix.lock'
+type mixLockAnalyzer struct{}
+
+func (a mixLockAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInput) (*analyzer.AnalysisResult, error) {
+	p := mix.NewParser()
+	res, err := language.Analyze(types.Hex, input.FilePath, input.Content, p)
+	if err != nil {
+		return nil, xerrors.Errorf("%s parse error: %w", input.FilePath, err)
+	}
+	return res, nil
+}
+
+func (a mixLockAnalyzer) Required(_ string, fileInfo os.FileInfo) bool {
```

Comment:
I prefer using `filePath` so tests can be simple.

Reply:
Okay. Thank you for changing this!